app/scripts/modules/core/serialize/serialize.write.service.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.serialize.write.service', [
+    require('../task/taskExecutor.js'),
+    require('../utils/lodash.js'),
+    require('../account/account.service.js')
+  ])
+  .factory('serializeWriter', function($q, taskExecutor, accountService, _) {
+
+    let serializationProviders = ['gce'];
+
+    function buildJobs(app, accountDetails) {
+      let jobs = [];
+      accountDetails.forEach((accountDetail) => {
+        if (serializationProviders.includes(accountDetail.cloudProvider)) {
+          jobs.push({
+            type: 'serializeApplication',
+            credentials: accountDetail.name,
+            applicationName: app.name,
+            cloudProvider: accountDetail.cloudProvider,
+          });
+        }
+      });
+      return jobs;
+    }
+
+    function loadAccountDetails(app) {
+      let accountDetailPromises = [];
+      let accounts = _.isString(app.accounts) ? app.accounts.split(',') : [];
+      accounts.forEach((account) => {
```

Comment:
`.map` would be cleaner here:
`let accountDetailPromises = accounts.map(account => accountService.getAccountDetails(account));`